### DISCUSSION TITLE
How do i put the meetings on the main menu?

### DISCUSSION THREAD

**Discussion by troybon** [code4recovery/12-step-meeting-list#492] [Q&A] [answered]
Hello, 

I am trying to figure out how to get the meetings page on the main menu.  I am using DIVI for my theme and builder. 

Thanks in advance.  

Troy

**Comment by joshreisner** [comment, marked as answer]
Hi Troy,

We don't offer support for specific themes, but generally speaking you can do it in Appearance > Menus or Appearance > Customize > Menus, the latter is shown here.

You can ususually add a new item, and select > Meetings > Post type archive, or you could add a custom link like `/meetings` (or whatever the proper link is in your case). You can get the link to your meetings page from Meetings > Import & Settings > Where's My Info?

![Screen Shot 2021-08-04 at 9 04 53 AM](https://user-images.githubusercontent.com/1551689/128215254-b5869796-caa5-4b6f-9fff-a9951bc8b46b.png)
